### ISSUE TITLE
Retracted patches documentation

### ISSUE THREAD

**Issue by hustodemon** [uyuni-project/uyuni-docs#935]
RFC: https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00074-retracted-updates-support.md

## Place
I tried to look for a best place where to put the new information, but couldn't find any (looked in the reference guide and in the administration guide). Therefore I'm proposing creating a new page under the Administration guide with the following contents:

==== BEGINNING OF PROPOSED SECTION

The Patch object in Uyuni has an "advisory status" attribute, but until now, we did not use it. Some time ago, SCC introduced a new value for this attribute: "retracted" and now  we make use of this attribute in Uyuni. A patch is "retracted", when its advisory status attribute is set to "retracted". A package is "retracted", when it belongs to a "retracted" patch.

The vendor (SCC) can retract an already released patch in case of disaster (the patch could break user systems). Uyuni syncs this information when running reposync on the vendor channels. Retracting patches only happens rarely.

A retracted patch/package cannot be installed on systems via Uyuni. The only way to override this behavior, is to install the package belonging to a retracted patch manually using zypper:

```bash
zypper my-retracted-package-1.0.0 # the exact version
```

Retracted patches/packages are depicted with an icon in relevant pieces of UI in Uyuni (e.g. list of channel patches/packages).

When a patch/package is installed on a system and becomes retracted afterwards, an icon will be displayed for affected packages/patches in the list of installed packages/patches on that system.

### Using channel clones
Upon cloning vendor channels into your organization, patches will be cloned as well. 
When the vendor retracts a patch in a channel and Uyuni synchronizes this channel (reposync), the "retracted" attribute will not get propagated to the cloned patches. Clients subscribed to the cloned channel will not see this. In order to propagate the attribute to your cloned channels use:

- either patch sync function (`Software -> Manage -> <select channel> -> Patches -> Sync`)
- or use Content Lifecycle Management, for more info check docs (insert a link to the content lifecycle management section I've added in this PR: https://github.com/uyuni-project/uyuni-docs/pull/899)

#### Patch sharing
Important thing: when you create multiple vendor channel clones in your organization, the patches are not cloned multiple times, but are shared between cloned channels. When you sync your cloned patch (either using the patch sync function or with Content Lifecycle Management), all channels using the cloned patch will observe that change.

==== END

## Documentation Update Required

We've implemented the "retracted patches" awareness in Uyuni recently and it should be documented.

## What does the documentation say now?

No existing docs.

## What change is required?

See above.

## Subject Matter Expert (SME)

@hustodemon 
@mcalmer 
@lucidd 

## Version

- Uyuni
- SUSE Manager 4.2 GA
- SUSE Manager 4.1

**Comment by hustodemon** [comment]
@keichwa Please take a look and ping me later. We need to agree on where we put the documentation.

@mcalmer Could you please also check this issue quickly? I tried to extract the minimum most important parts of the "feature".

**Comment by keichwa** [comment]
Yes, a new chapter in the Admin Guide seems to be the best fit.  Title proposals: "Manage Patches" or "Patch Management".  I unfortunately miss that one until now.  Thanks for putting the info together.  Can you turn it into a PR?  This would be great.

**Comment by hustodemon** [comment]
Ok, taking this over.
